charts/vald-helm-operator/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for vald-helm-operator
+name: vald-helm-operator
+version: v0.0.25
+keywords:
+  - Vald
+  - NGT
+home: https://vdaas.org
+# icon:
```

Comment:
maybe we can make icon could you please contact to @taisuou he is our designer

Reply:
just added that one. :+1:

Reply:
replaced it with whitespace trimmed version.

Reply:
It seems not square, is it ok?

Reply:
maybe it's ok.
there's no specification about size or shape in Helm docs.

Reply:
I see, thank you